engines/wintermute/detection_tables.h

```diff
@@ -991,6 +1006,14 @@ static const WMEGameDescription gameDescriptions[] = {
 		WME_ENTRY2s("german.dcp", "bf4c2b8c26342342441a6d64934ab832", 107027865,
 					"data.dcp", "50de0beaa5ad621aa9f020df901d1e74", 1342214), Common::DE_DEU, ADGF_UNSTABLE, WME_1_9_1),
 
+	// Limbo of the Lost (English)
+	WME_WINENTRY("lotl", "",
```

Comment:
Is it a 2D or 2.5D game? I don't have it, but it looks 2.5D on youtube.
In this case we need "NOTE: This is a 2.5D game that is out of ScummVM scope" here.

Reply:
You're totally right.. I'll fix it

Reply:
The same applies to "wmedemo3d".

Reply:
Niche catches! I overlooked them!